application/.gitignore

```diff
@@ -0,0 +1,7 @@
+coverage.xml
```

Comment:
We have lots of gitignores and they look broadly the same ... can we get one that covers all sub-directories? Is that better ?